quickinstall.py

```diff
@@ -104,9 +104,9 @@ def do_install(self):
         if pip_version < [6, 0]:
             args += ['--download-cache', self.download_cache, ]
         if pip_version >= [1, 5]:
-            args += [
-                '--process-dependency-links',
-            ]
+            args += ['--process-dependency-links', ]
+        if pip_version >= [9, 0]:
+            args += ['--upgrade-strategy=eager', ]
```

Comment:
https://github.com/pypa/pipenv/issues/438 9.0 or 10.0?

Reply:
https://github.com/girder/girder/pull/2695 

manthey commented on Apr 17

pip 9.x honors the flag. pip 8 fails if it is specified.

jbeezley commented on Apr 17

The flag was added in 9, the default was changed in 10.
--